registry/quilt_server/views.py

```diff
@@ -2339,3 +2431,38 @@ def comments_list(owner, package_name):
     comments = Comment.query.filter_by(package=package).order_by(Comment.created)
 
     return dict(comments=map(_comment_dict, comments))
+
+IDENTITY_POOL_ID = 'us-east-1:bad5e611-6efb-4b71-8f46-674fa56901b8'
+AWS_TOKEN_DURATION = 60 * 60 # 60 minutes
+
+@app.route('/api/auth/get_credentials', methods=['GET'])
+@api(require_login=True)
+@as_json
+def aws_token():
+    client = boto3.client('cognito-identity')
```

Comment:
Need to pass credentials here, the way we're doing it for the `s3_client`. Don't want to accidentally access the real thing while running unit tests.
(And it should probably be a global, too.)